sdk/core/Azure.Core/src/RequestContent.cs

```diff
@@ -72,7 +76,7 @@ public abstract class RequestContent : IDisposable
         /// </summary>
         /// <param name="content">The <see cref="BinaryData"/> to use.</param>
         /// <returns>An instance of <see cref="RequestContent"/> that wraps a <see cref="BinaryData"/>.</returns>
-        public static RequestContent Create(BinaryData content) => new MemoryContent(content.ToMemory());
+        public static new RequestContent Create(BinaryData content) => new MemoryContent(content.ToMemory());
```

Comment:
Do we need `new` modifier for static method?

Reply:
Yes - otherwise this method hides the method on the base type in order to change the return type and will not compile.

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-net/pull/42226